docs/military/townsville.md

```diff
@@ -4,7 +4,82 @@
+
+## Airspace
+### TCU
+TL TCU owns the Class C and G airspace within 36 DME TL from `SFC` to `FL180`
+
+<figure markdown>
+![TL TCU/ADC Airspace Structure](img/tltcu.png){ width="2000" }
+  <figcaption>TL ADC/TCU Airspace Structure</figcaption>
+</figure>
+
+### ADC
+TL ADC owns the Class C arispace in the TL CTR (Extends to 7NM of the thresholds of 01 and 19.) from `SFC` to `A015`. This airsapce is designed to facilitate Helicopter Traffic and Ciruit Traffic.
+
+### Surveillance
+TL ADC is permitted to use Surveillance standards for separation. Surveillance coverage can be expected to be available at all levels in the TL ADC arispace. 
+For simulations purposes,  visual separation is assumed to exist belwo the cloud base, and whtin 7NM of the areodrome. Visual separation can still be used to separate from aircraft on an instrument approach, below the cloud base.
+
+## Aerodrome
+### Runway Operations
+Runways 01 and 19 are the primary runways at Townsville. 
+Runway 07 is often used in conjunction with 01 for VFR arrivals and Helicopter Traffic. 
+
+### NON-RNAV Departures
+For non-RNAV approved IFR aircaft with a wake turbulence category of medium or greater, issue a RADAR SID.
+For non-RNAV approved IFR aircraft with a wake turbulence category of light, issue a visual departure or Radar SID
+
+## Charts
 !!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## Coordination
+### Enroute
+#### Departures
+The Standard assignable level from TL TCU to TBP is the lower of `FL180` or the `RFL`. All other aircraft must be voice coordinated. 
+
+!!! example
+    <span class="hotline">**TL TCU** -> **TBP**</span>: "JST729, with your concurrence, will be assigned `FL160`, for my separation with QLK327D"  
+    <span class="hotline">**TBP** -> **TL TCU**</span>: "JST729, concur `FL160`"  
+
+#### Arrivals
+The Standard assignable level from KEN(TBP) to TL TCU is `A100`, and assigned a STAR. All other aircraft must be voice coordinated.
+
+### TL ADC
+#### Auto Release
+"Next" coordination is **not** required to TL TCU for aircraft that are:  
+a) Departing from a runway nominated on the ATIS; and  
+b) Assigned the standard assignable level; and  
+c) Assigned a Procedural SID  
+
+All other aircraft require a "Next" call to TL TCU.
+
+!!! example
+    <span class="hotline">**DN ADC** -> **DN TCU**</span>: "Next, DNGO05, runway 19"  
+    <span class="hotline">**DN TCU** -> **DN ADC**</span>: "DNGO05, Assigned Heading Left 150, unrestricted"  
+    <span class="hotline">**DN ADC** -> **DN TCU**</span>: "Assigned Heading Left 150, DNGO05"  
+
+### Miscellaneous
```

Comment:
Move Miscellaneous above Coordination section to be in line with other SOP pages